COMETwebapp/Components/SubscriptionDashboard/CompoundParameterEvolution.razor.cs

```diff
@@ -0,0 +1,121 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+//  <copyright file="CompoundParameterEvolution.razor.cs" company="RHEA System S.A.">
+//     Copyright (c) 2023 RHEA System S.A.
+// 
+//     Author: Sam Gerené, Alex Vorobiev, Alexander van Delft, Jaime Bernar, Théate Antoine, Nabil Abbar
+// 
+//     This file is part of COMET WEB Community Edition
+//     The COMET WEB Community Edition is the RHEA Web Application implementation of ECSS-E-TM-10-25 Annex A and Annex C.
+// 
+//     The COMET WEB Community Edition is free software; you can redistribute it and/or
+//     modify it under the terms of the GNU Affero General Public
+//     License as published by the Free Software Foundation; either
+//     version 3 of the License, or (at your option) any later version.
+// 
+//     The COMET WEB Community Edition is distributed in the hope that it will be useful,
+//     but WITHOUT ANY WARRANTY; without even the implied warranty of
+//     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//    Affero General Public License for more details.
+// 
+//    You should have received a copy of the GNU Affero General Public License
+//    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//  </copyright>
+//  --------------------------------------------------------------------------------------------------------------------
+
+namespace COMETwebapp.Components.SubscriptionDashboard
+{
+    using CDP4Common.EngineeringModelData;
+    using CDP4Common.SiteDirectoryData;
+    using CDP4Common.Types;
+
+    using COMETwebapp.ViewModels.Components.SubscriptionDashboard.Rows;
+    
+    using DevExpress.Blazor;
+    
+    using Microsoft.AspNetCore.Components;
+
+    /// <summary>
+    /// Component that display the evolution of a <see cref="ParameterSubscriptionRowViewModel" /> for a <see cref="CompoundParameterType" />
+    /// </summary>
+    public partial class CompoundParameterEvolution
+    {
+        /// <summary>
+        /// The <see cref="ParameterSubscriptionRowViewModel" /> to display the evolution
+        /// </summary>
+        [Parameter]
+        public ParameterSubscriptionRowViewModel ParameterSubscriptionRow { get; set; }
+
+        /// <summary>
+        /// The name of selected dimension
+        /// </summary>
+        private string selectedDimension = "1";
```

Comment:
If the dimensions have a define set I propose to use an enum with the possible values instead of a magic string

Reply:
selectedDimension is the dimension the user chooses in order to visualize the revision evolution of each parameter in this dimension. It take 1 as default value